tests/json-fixtures/test_virtual_machine.py

```diff
@@ -187,7 +188,8 @@ def test_vm_fixtures(fixture, vm_class, computation_getter):
         gas_limit=fixture['env']['currentGasLimit'],
         timestamp=fixture['env']['currentTimestamp'],
     )
-    vm = vm_class(header=header, chaindb=chaindb)
+    chain_context = ChainContext(chain_id=42)  # TODO is there a better way to do this?
```

Comment:
IIUC, this will have to partially be answered by the test fixture builders. They will have to supply a chainID in the `env` if they want to test different ones. But presumably we need to handle the default case where nothing is supplied. Choosing an arbitrary chain ID 42 when nothing is supplied seems fine to me.

Reply:
I think the value where nothing is supplied should be `0` or `None` because 42 is actually a valid chain ID for the Kovan network: https://chainid.network/

Reply:
Since it's not defined for `None`, chain ID should probably use some actual integer. Zero seems fine, or something random and a bit big, like `0xa1fa1fa = 169845242`

Reply:
Currently, pre-`EIP-155` tranasctions (and actually post too, they're still valid) does not have to define a chain ID since one didn't exist. How is this handled?

Reply:
Also, 0 is probably the best choice since I don't think it is defined as being valid with 0 as the value.

Reply:
> Currently, pre-`EIP-155` tranasctions (and actually post too, they're still valid) does not have to define a chain ID since one didn't exist. How is this handled?

How do you mean? `EIP-155` isn't validated at all, yet.

Reply:
oh, I think I understand now why you made this comment. `chain_id` is a mandatory parameter of `ChainContext`, I just used the kwarg syntax to make it clearer what that number was for